init-tools.cmd

```diff
@@ -1,6 +1,9 @@
 @echo off
 setlocal
 
+REM Workaround https://github.com/dotnet/coreclr/issues/2153
+set ComPlus_ReadyToRun=0
```

Comment:
What about outside of this script? When you build leave node projects it will still run dotnet restore. Do folks need to set this for that command? Or is it just dotnet publish? Perhaps we need to document this as a workaround for now.